### DISCUSSION TITLE
Not getting any new instances

### DISCUSSION THREAD

**Discussion by RohitKS7** [OpenZeppelin/ethernaut#506] [Support]
Hlo, Friends!
I don't know if it's a blockchain problem or game's problem but
After clicking "Get New Instance", It will sent a transaction but even after 10 mins its in pending state. So, I can't able to play Ethernaut!

**Comment by xaler5** [comment]
Hey @RohitKS7 have you tried checking the gas used ? probably the transaction is getting stuck in the mempool. You can even try chaing the network to another one or using more gas to speedup the transaction
